src/models/CommonModel.ts

```diff
@@ -165,6 +167,7 @@ export class CommonModel extends CommonSchema<CommonModel> {
       } else {
         for (const [propName, prop] of Object.entries(mergeFromProperties)) {
           if (mergeToProperties[`${propName}`] !== undefined) {
+            Logger.warn(`Found duplicate properties ${propName} for model. Model property from ${mergeFrom.$id || 'unknown'} merged into ${mergeTo.$id || 'unknown'}`, mergeTo, mergeFrom, originalSchema);
```

Comment:
It also should be an info.

Reply:
see https://github.com/asyncapi/generator-model-sdk/pull/184#discussion_r632957284